src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -218,12 +212,6 @@ public async Task<bool> SaveServerExclusionsAsync(CancellationToken cancellation
             return true;
         }
 
-        public void InstallPackages(IProgress<FixedStepsProgress> progress, CancellationToken cancellationToken)
-        {
-            // TODO - CM cleanup
-            this.InternalState.BindingOperationSucceeded = true;
```

Comment:
The initials state of `BindingOperationSucceeded` is `true`, so removing this didn't have any effect.

All of the remaining tests pass, and I also manually bound and updated a project to check it suceeded.